generator/vogtle_data_generator.py

```diff
@@ -58,49 +58,168 @@ def __init__(self, directory=None):
+
+    def generate_itaac_efforts(self, itaac_ids):
+        """Generate ITAAC Efforts
+
+        :param itaac_ids: IDs of ITAACS to generate efforts
+        """
+        effort_types = [
+            "ITAAC",
+            "Program",
+            "Reactive/Allegations",
+            "Technical",
+            "Total",
+        ]
+        output = []
+
+        with open(f'{self.directory}itaac_efforts.csv', 'w+') as output_file:
+
+            for idx, itaac_id in enumerate(itaac_ids, 1):
+                actual = 0
+                estimated = 0
+
+                for effort_type in effort_types:
+                    if effort_type == 'Total':
+                        output.append(f'\n{idx}|'
+                                      f'{itaac_id}|'
+                                      f'{effort_type}|'
+                                      f'{actual}|'
+                                      f'{estimated}')
+                    else:
+                        actual += random.randint(0, 120)
+                        estimated += random.randint(0, 120)
+
+            output_file.write("id|itaac_id|effort_type|actual|estimate")
+            output_file.write(''.join(output)
+        """Generate ITAAC Efforts
+
+        :param itaac_ids: IDs of ITAACS to generate efforts
+        """
+        effort_types = [
+            "ITAAC",
+            "Program",
+            "Reactive/Allegations",
+            "Technical",
+            "Total",
+        ]
+        output = ["id|itaac_id|effort_type|actual|estimate"]
+        effort_types = ["ITAAC", "Program", "Reactive/Allegations",
+                        "Technical", "Total"]
+
+        with open('{}itaac_efforts.csv'
+                  .format(self.directory), 'w+') as output_file:
+
+            # write header
+            for effort_id in range(1, len(itaac_ids)+1):
+                actual_total = 0
+                estimated_total = 0
+
+                itaac_id = itaac_ids[effort_id - 1]
+                for effort_type in effort_types:
+
+                    if effort_type == 'Total':
+                        actual = actual_total
+                        estimate = estimated_total
+                    else:
+                        actual = random.choice(list(range(0, random.choice(
+                            [40, 60, 80, 100, 120]))))
+                        estimate = random.choice(list(range(0, random.choice(
+                            [40, 60, 80, 100, 120]))))
+
+                        actual_total += actual
+                        estimated_total += estimate
+
+                    output.append("\n%s|%s|%s|%s|%s" % (effort_id,
+                                                        itaac_id,
+                                                        effort_type,
+                                                        actual,
+                                                        estimate))
+
+            output_file.write(''.join(output))
```

Comment:
@calderonmluis The github "Suggestions" feature cannot be applied to multiline comments, so I put it on the first line of the function.  When you accepted the suggestion only the first line of the previous code was replaced.  Would you please remove lines 114-158.  Sorry about that.

Reply:
I noticed and am working on the fix locally. Also, should I rebase? I can just pull in the changes from `develop`, is there any reason you prefer a rebase instead?

Reply:
@calderonmluis I think the only main difference between a rebase and a merge, in this case, is the history.  With a rebase it is very clear where a branch was on the trunk and what commits preceded it.

The tree will look like this:

![Screen Shot 2019-12-31 at 8 09 51 AM](https://user-images.githubusercontent.com/7132188/71625471-3ced3c00-2ba5-11ea-9072-7961c6056d21.png)


I normally rebase a feature branch before submitting a PR and after incorporating any comments from reviewers.  If you rebase during a review all the code shows up as "changed" making it hard on the reviewers.

```bash
git pull --rebase origin develop
```

A rebase before submitting the PR lets me see any merge conflicts that need to be addressed.

A rebase after the review ensures another PR wasn't merged that causes me trouble.  This is also a great time to consolidate fixup commits that were made for reviewers.

### An example where O represents a commit:
O ENH: new function 1
O ENH: new function 2
git pull --rebase origin develop
O FIX: update function 1
Rebase FIX into ENH: new function 1
Submit PR
O FIX: address xyz issue for function 1 from reviewer 3
O FIX: address mno issue for function 2 from reviewer 5
Review is complete
git pull --rebase origin develop
Rebase FIX commits into original ENH commits
Merge PR

Reply:
I went ahead and ran the rebase. Thanks for the approval